docs/tutorial/debugging-main-process-vscode.md

```diff
@@ -35,3 +39,49 @@ $ code electron-quick-start
 Set some breakpoints in `main.js`, and start debugging in the [Debug View](https://code.visualstudio.com/docs/editor/debugging). You should be able to hit the breakpoints.
 
 Here is a pre-configured project that you can download and directly debug in VSCode: https://github.com/octref/vscode-electron-debug/tree/master/electron-quick-start
+
+## Native C++ (Windows)
```

Comment:
Can we add an introduction paragraph to this debugging section that explains that step 0 is to get a working checkout of the Electron codebase (and maybe link the relevant docs + build-tools)? People reading through the docs might be confused about where to get started and what the purpose of this section is if they aren't already familiar with building Electron from source.

Reply:
That makes sense. I can add an intro and link to [these build instructions](https://www.electronjs.org/docs/development/build-instructions-gn), as it goes through building from source and includes a link to build-tools in the beginning. I've only built using build-tools, so these are the only relevant/helpful links I can think of, but I can add any more I can find.

Reply:
I added a brief intro in the latest commit.